Form: describe Smart Paste API (#8011)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxForm/1 Configuration/aiIntegration.md

---
---
##### shortDescription

> Binds the AI service to the Form.

Reviewer: The issues I see here are the order of things and articles. So this is a property of the form class. So I can bind **the form** (where I specify this property) to **a service** (I assume I can specify any service that I want). 

In the same way, I would bind "the grid to a data source". I wouldn't bind "the data source to the grid". 

```suggestion
Binds the Form to an AI service.
```

---

> Use the following members to activate AI-powered **Smart Paste** functionality for the Form component:

Reviewer 2: ```suggestion
Use the following members to activate the AI-powered **Smart Paste** functionality for the Form component:
```

> The Form component ships with the **Smart Paste** functionality. To activate this capability, specify:

Reviewer: ```suggestion
Use the following members to activate AI-powered **Smart Paste** functionality for the Form component:
```

> - *`aiIntegration`* - accepts an [AIIntegration](/api-reference/40%20Common%20Types/AIIntegration '/Documentation/ApiReference/Common_Types/AIIntegration/') object that holds AI Service settings.

Reviewer 2: ```suggestion
- *`aiIntegration`* - accepts an [AIIntegration](/api-reference/40%20Common%20Types/AIIntegration '/Documentation/ApiReference/Common_Types/AIIntegration/') object that stores AI Service settings.
```

> - [AIIntegration](/api-reference/40%20Common%20Types/AIIntegration '/Documentation/ApiReference/Common_Types/AIIntegration/') class
> - *'smartPaste'* button [name](/Documentation/ApiReference/UI_Components/dxForm/Types/#FormPredefinedButtonItem) (or call the [smartPaste(text)](/Documentation/ApiReference/UI_Components/dxForm/Methods/#smartPastetext) method)

Reviewer: I want the list to be more consistent. Right now, you list a class in the first item (you must assign that class to the property, as I understand). The second item starts with what I believe is a property name. 

I would be better if both items start with the property name.

```suggestion
- *`aiIntegration`* - accepts an [AIIntegration](/api-reference/40%20Common%20Types/AIIntegration '/Documentation/ApiReference/Common_Types/AIIntegration/') object that holds AI Service settings.
- *'smartPaste'* - specifies a button [name](/Documentation/ApiReference/UI_Components/dxForm/Types/#FormPredefinedButtonItem). (You can also call the [smartPaste(text)](/Documentation/ApiReference/UI_Components/dxForm/Methods/#smartPastetext) method.)
```
